src/shared/utils/number-to-US-locale.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ * This file is part of Neo4j.
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+export default value =>
+  (parseInt(value, 10) && parseInt(value, 10).toLocaleString('en-US')) || value
```

Comment:
thought process here is to be defensive against anything where `parseInt` returns `NaN` and just return the value passed in as it's a separate issue. Any other ways to do this or suggestions or comments to be more in in line with Neo4j patterns is welcome

Reply:
That looks good to me